apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -55,19 +54,18 @@ const IdeaForm = ({
 						style={{
 							maxWidth: '36rem',
 							margin: '1.5rem auto 1.5rem',
-							padding: '2rem',
 							textAlign: 'left',
-							borderRadius: 32,
-							backgroundColor: 'white',
 						}}
 					>
 						<AutoSubmitToken />
 						<atoms.Typography type='h4' style={{
 							maxWidth: '37rem',
 							margin: '1rem auto 2.5rem',
 							textAlign: 'left',
+							fontSize: '1.5rem',
```

Comment:
Is it necessary to change the `fontSize` here? I'm just wondering because this is a component from the design team